src/splits.rs

```diff
@@ -299,6 +299,318 @@ pub enum Split {
+    /// Splits after rescuing flea in Song_14
+    SavedFleaSong14,
+    /// Rescued Flea Underworks Explosions (Flea)
+    ///
+    /// Splits after rescuing flea in Under_21
+    SavedFleaUnder21,
+    /// Rescued Flea Underworks Wisp Thicket (Flea)
+    ///
+    /// Splits after rescuing flea in Under_23
+    SavedFleaUnder23,
+    /// Defeated Giant Flea (Flea)
+    ///
+    /// Splits after defeating Giant Flea
+    TamedGiantFlea,
+    /// Met Vog (Flea)
+    ///
+    /// Splits after talking to Vog
+    MetTroupeHunterWild,
+    /// Freed Kratt (Flea)
+    ///
+    /// Splits after freeing Kratt
+    CaravanLechSaved,
+    // endregion: FleaSpecific
+
+    // region: Bellways
+    /// Unlocked Putrified Ducts (Bellway)
+    ///
+    /// Splits after unlocking Putrified Ducts Bellway
+    UnlockedAqueductStation,
+    /// Unlocked Bellhart (Bellway)
+    ///
+    /// Splits after unlocking Bellhart Bellway
+    UnlockedBelltownStation,
+    /// Unlocked Far Fields (Bellway)
+    ///
+    /// Splits after unlocking Far Fields Bellway
+    UnlockedBoneforestEastStation,
+    /// Unlocked Grand Bellway (Bellway)
+    ///
+    /// Splits after unlocking Grand Bellway
+    UnlockedCityStation,
+    /// Unlocked Blasted Steps (Bellway)
+    ///
+    /// Splits after unlocking Blasted Steps Bellway
+    UnlockedCoralTowerStation,
+    /// Unlocked Deep Docks (Bellway)
+    ///
+    /// Splits after unlocking Deep Docks Bellway
+    UnlockedDocksStation,
+    /// Unlocked Greymoor (Bellway)
+    ///
+    /// Splits after unlocking Greymoor Bellway
+    UnlockedGreymoorStation,
+    /// Unlocked Mount Fay (Bellway)
+    ///
+    /// Splits after unlocking Mount Fay Bellway
+    UnlockedPeakStation,
+    /// Unlocked Bilewater (Bellway)
+    ///
+    /// Splits after unlocking Bilewater Bellway
+    UnlockedShadowStation,
+    /// Unlocked Shellwood (Bellway)
+    ///
+    /// Splits after unlocking Shellwood Bellway
+    UnlockedShellwoodStation,
+    // endregion: Bellways
+
+    // region: Ventricas
+    /// Unlocked Choral Chambers (Ventrica)
+    ///
+    /// Splits after unlocking Choral Chambers Ventrica
+    UnlockedSongTube,
+    /// Unlocked Underworks (Ventrica)
+    ///
+    /// Splits after unlocking Underworks Ventrica
+    UnlockedUnderTube,
+    /// Unlocked Grand Bellway (Ventrica)
+    ///
+    /// Splits after unlocking Grand Bellway Ventrica
+    UnlockedCityBellwayTube,
+    /// Unlocked High Halls (Ventrica)
+    ///
+    /// Splits after unlocking High Halls Ventrica
+    UnlockedHangTube,
+    /// Unlocked Songclave (Ventrica)
+    ///
+    /// Splits after unlocking Songclave Ventrica
+    UnlockedEnclaveTube,
+    /// Unlocked Memorium (Ventrica)
+    ///
+    /// Splits after unlocking Memorium Ventrica
+    UnlockedArboriumTube,
+    // endregion: Ventricas
+
+    // region: ShakraEncounters
+    /// Seen Shakra Bonebottom (Mapper)
+    ///
+    /// Splits after seeing Shakra in Bonebottom
+    SeenMapperBonetown,
```

Comment:
In the interest of creating less work for Silksong Split Maker, I'd like all these `(Mapper)` parentheticals changed to `(NPC)`